packages/tutorial/src/part6.tsx

```diff
@@ -0,0 +1,124 @@
+import * as AI from 'ai-jsx';
+import { ChatCompletion, SystemMessage, UserMessage } from 'ai-jsx/core/completion';
+import { Route, NaturalLanguageRouter } from 'ai-jsx/batteries/natural-language-router';
+import { showInspector } from 'ai-jsx/core/inspector';
+import enquirer from 'enquirer';
+
+const { prompt } = enquirer;
+
+/** Prompt the user for their question. */
+async function getQuestion(questionNumber: number, maxQuestions: number) {
+  const response = await prompt({
+    type: 'input',
+    name: 'question',
+    message: `[Question ${questionNumber + 1}/${maxQuestions}] Your question`,
+  });
+  // @ts-ignore
```

Comment:
```suggestion
  // @ts-expect-error
```

We always want to use `ts-expect-error`. That way, if the underlying error goes away, we get a warning, and can remove the comment.